tests/plugins/cell.test.ts

```diff
@@ -64,23 +64,38 @@ describe("link cell", () => {
     (url) => {
       const model = new Model();
       setCellContent(model, "A1", `[my label](${url})`);
-      const cell = getEvaluatedCell(model, "A1");
+      let cell = getEvaluatedCell(model, "A1");
       expect(cell.link?.label).toBe("my label");
       expect(cell.link?.url).toBe(url);
       expect(urlRepresentation(cell.link!, model.getters)).toBe(url);
       expect(getCell(model, "A1")?.content).toBe(`[my label](${url})`);
       expect(getStyle(model, "A1")).toEqual({ textColor: LINK_COLOR });
       expect(getCellText(model, "A1")).toBe("my label");
+
+      setCellContent(model, "B1", `=HYPERLINK("${url}", "Odoo")`);
```

Comment:
move to another test. (same for the other tests)
Tests should be small and test one thing only.
This existing one tests markdown content, yours test HYPERLINK content